actionview/lib/action_view/helpers/tag_helper.rb

```diff
@@ -46,8 +47,180 @@ class TagBuilder # :nodoc:
+              end
+            end
+          RUBY
+        end
+
+        def self.define_self_closing_element(name, **options)
+          define_void_element(name, self_closing: true, **options)
+        end
+
+        define_void_element :area
+        define_void_element :base
+        define_void_element :br
+        define_void_element :col
+        define_void_element :embed
+        define_void_element :hr
+        define_void_element :img
+        define_void_element :input
+        define_void_element :keygen
+        define_void_element :link
+        define_void_element :meta
+        define_void_element :param
+        define_void_element :source
+        define_void_element :track
+        define_void_element :wbr
+
+        define_self_closing_element :animate
+        define_self_closing_element :animateMotion
+        define_self_closing_element :animateTransform
+        define_self_closing_element :circle
+        define_self_closing_element :ellipse
+        define_self_closing_element :line
+        define_self_closing_element :path
+        define_self_closing_element :polygon
+        define_self_closing_element :polyline
+        define_self_closing_element :rect
+        define_self_closing_element :set
+        define_self_closing_element :stop
+        define_self_closing_element :use
+        define_self_closing_element :view
+
+        define_element :a
+        define_element :abbr
+        define_element :acronym
+        define_element :address
+        define_element :article
+        define_element :aside
+        define_element :audio
+        define_element :b
+        define_element :bdi
+        define_element :bdo
+        define_element :big
+        define_element :blockquote
+        define_element :body
+        define_element :button
+        define_element :canvas
+        define_element :caption
+        define_element :center
+        define_element :cite
+        define_element :code
+        define_element :colgroup
+        define_element :data
+        define_element :datalist
+        define_element :dd
+        define_element :del
+        define_element :details
+        define_element :dfn
+        define_element :dialog
+        define_element :dir
+        define_element :div
+        define_element :dl
+        define_element :dt
+        define_element :em
+        define_element :fieldset
+        define_element :figcaption
+        define_element :figure
+        define_element :font
+        define_element :footer
+        define_element :form
+        define_element :frame
+        define_element :frameset
+        define_element :h1
+        define_element :h2
+        define_element :h3
+        define_element :h4
+        define_element :h5
+        define_element :h6
+        define_element :head
+        define_element :header
+        define_element :hgroup
+        define_element :html
+        define_element :i
+        define_element :iframe
+        define_element :image
```

Comment:
While I've copied this list entirely from MDN, there are some elements that are either non-standard, deprecated, or both.

For example, the [`<image>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/image) is marked as non-standard and deprecated.

Since our plan is to generate RDoc methods from comments on each of these, would it be worth going through this list and omitting and element that is deprecated or non-standard?

Applications calling `tag.image` will still be able to render `<image>` elements (and gain the benefits of an `#image` method the next time they call it), but it won't be part of the object's public API.